### PR TITLE
Removes Ling team objectives

### LINKED ISSUE #589: Changeling group goals can get the objective to become a IPC
title.

1: Ensure changelings impersonate and escape as the following members of the medical department: Ark Shaffer the Virologist, Sarah Banlon the Medical Doctor, FENIX-2496 the Geneticist, John Sweeps the Geneticist, while the real members are dead. This is a team objective. 

***FENIX-2496 the Geneticist,***

### PR DESCRIPTION
**Pull request by Ziiro**
:cl: 
tweak:  Removes ling team objectives. Trust no one!
/:cl:

I'm not 100% I got everything removed properly here, definitely want someone else to look over this.

[why]: Team ling objectives are a particular thorn for a two reasons:
1) Either all of your ling teammates are bad, and the objective becomes pointless because you'll never greentext it
2) OR the opposite happens, where the Lings actually succeed and then take control of the station. Which generally leads to murderboning.

Also fixes https://github.com/OracleStation/OracleStation/issues/589 (in a roundabout way)

### CODE REVIEW

File: code/game/gamemodes/changeling/changeling.dm [hunk 1]
```diff
@@ -178,24 +151,14 @@ GLOBAL_LIST_INIT(slot2type, list("head" = /obj/item/clothing/head/changeling, "w
...
 /datum/game_mode/changeling/forge_changeling_objectives(datum/mind/changeling)
-	if(changeling_team_objective_type)
-		var/datum/objective/changeling_team_objective/team_objective = new changeling_team_objective_type
-		team_objective.owner = changeling
-		changeling.objectives += team_objective
-
-		..(changeling,1)
-	else
-		..(changeling,0)
+	..(changeling,0)
```
Comment (Ziiro) [inline code/game/gamemodes/changeling/changeling.dm]: wait, this is pointless. 🤔

**Comment by ike709** [comment]
now just remove lings entirely and we're golden

**Comment by scrubmcnoob** [comment]
There's a reason lings get code names in hive chat. Team lings should only occur through player's own initiative where there's still a chance the lings can fuck each other over.

**Comment by Ralta** [comment]
I much prefer lings that fuck each other over.

Out of curiosity, is there anything that currently prevents lings getting each other as targets?

**Review by AndrewMontagne — APPROVED** [review]
Code checks out, up to the balance police to decide whether it gets merged.

**Comment by FlattestGuitar** [comment]
I'm pretty sure we've agreed to merge this before. Revert if no bueno.

**Pull request merged**